galgebra/ga.py

```diff
@@ -1276,15 +1276,11 @@ def Mul(self, A, B, mode='*'):  # Unifies all products into one function
             raise ValueError('Unknown multiplication operator {!r}', mode)
 
     def mul(self, A, B):  # geometric (*) product of blade representations
-        if A == 0 or B == 0:
-            return 0
         return update_and_substitute(A, B, self.mul_table_dict)
 
     def wedge(self, A, B):
         # wedge assumes A and B are in blade rep
```

Comment:
Does this assumption still hold?

Reply:
Not sure - but `0` is both blade and base rep, so it won't make any difference

Reply:
If nothing else, the `^` operator always ensures this condition holds:

https://github.com/meuns/galgebra/blob/3fea69ff4c4ca8f8afea083b697ef9d5112824b9/galgebra/mv.py#L743-L747

Reply:
Not just for this PR, but for the correctness of comments after recent changes.

Reply:
Cool then.

Reply:
Probably a good callout though that `mul` does autoconversion from blade to base, but `wedge` does not do auto-conversion in the opposite direction.

Also raises the strange observation that `mul_table_dict` contains both blades and bases, which might not have been intended.